tests/models/opt/test_modeling_opt.py

```diff
@@ -38,11 +38,8 @@
 def prepare_opt_inputs_dict(
     config,
     input_ids,
-    decoder_input_ids=None,
```

Comment:
Hey @ArthurZucker,

I would also recommend reverting this as it's not related to OPT TF or Flax. Happy to correct it in a future PR :-)